src/rust/ensogl/src/system/gpu/data/uniform.rs

```diff
@@ -334,12 +334,16 @@ macro_rules! define_get_or_add_gpu_texture_dyn {
         , internal_format : AnyInternalFormat
         , item_type       : AnyItemType
         , provider        : P
+        , parameters      : Option<Parameters>
```

Comment:
Please never import names to scope which does not make sense here. What are uniform `Parameters` ? We should use names in qualified manner so the code is understandable. If Im in a file `uniform` and I see name `Parameters` it means "uniform parameters". This should be `texture::Parameters` instead.

Reply:
The texture module was already blanked imported in this file and used all over this place. Messing with this import is going to be a bit of a time sink.

Reply:
I bet it will not take more than couple of minutes :) If it was blanket imported then it was a bad design, but also I believe that no types which require qualification were used, only such as `Texture` which should be imported separately. Anyway, we should not merge a code which is confusing, and this one is, as `Uniform::Parameters` does not make sense.